server/src/fireboltOpenRpc.mjs

```diff
@@ -121,6 +125,10 @@ function getDeveloperNotesForMethod(methodName) {
 // Are the given params valid for thegiven method, based on the OpenRPC metadata?
 // Returns an array of errors; Returns an empty array if no errors are found
 function validateMethodCall(methodName, params) {
+  // Returns an empty array in "novalidate mode"
+  if( ! config.validate.includes("params") ){
```

Comment:
Same as above... I think no "!".

Reply:
I believe "!" is required here. 
if params is not present in config.validate, it means "--novalidate params" flag is set by user. 
if the flag is set, we shouldn't be validating params for given method and respond with empty array. That's what the method does currently. 

Does it make sense?

Reply:
This makes sense to me given the previous discussions. Correct me if I'm wrong that we're maintaining a list of things we need to validate. So "novalidate params" would remove "params" from the validate list.

Reply:
Correct!